Pcap++/header/PcapDevice.h

```diff
@@ -4,6 +4,10 @@
 #include "RawPacket.h"
 #include "PcapFilter.h"
 #include "PointerVector.h"
+

```

Comment:
Why are these lines necessary?

Reply:
Got it. To make it more clear I'd add a comment on above these lines explaining what you just explained here